manual-of-legal-citation.csl

```diff
@@ -0,0 +1,772 @@
+            <text value="版"/>
+          </if>
+          <else>
+            <label variable="locator" form="short"/>
+          </else>
+        </choose>
+      </if>
+      <else>
+        <text variable="locator"/>
+      </else>
+    </choose>
+  </macro>
+  <!-- 报纸的出版信息，一般注明年、月、日;必要时，注明版次。 -->
+  <macro name="locator-or-page">
+    <choose>
+      <if variable="locator">
+        <text macro="locator"/>
+      </if>
+      <else-if variable="page" type="article-newspaper" match="all">
+        <choose>
+          <if is-numeric="page">
+            <text value="第"/>
+            <text variable="page"/>
+            <text value="版"/>
+          </if>
+          <else>
+            <text variable="page"/>
+          </else>
+        </choose>
+      </else-if>
+    </choose>
+  </macro>
+  <macro name="locator-en">
+    <choose>
+      <if is-numeric="locator">
+        <choose>
+          <if type="article-newspaper">
+            <text term="at" suffix=" "/>
+            <text variable="locator" form="short"/>
+          </if>
+          <else>
+            <label variable="locator" form="short" suffix=" "/>
+            <text variable="locator"/>
+          </else>
+        </choose>
+      </if>
+      <else>
+        <text variable="locator"/>
+      </else>
+    </choose>
+  </macro>
+  <macro name="locator-or-page-en">
+    <choose>
+      <if variable="locator">
+        <text macro="locator-en"/>
+      </if>
+      <else-if variable="page">
+        <choose>
+          <if type="article-newspaper">
+            <text term="at" suffix=" "/>
+            <text variable="page" form="short"/>
+          </if>
+          <else-if type="article-magazine">
+            <label variable="page" form="short" suffix=" "/>
+            <text variable="page"/>
+          </else-if>
+        </choose>
+      </else-if>
+    </choose>
+  </macro>
+  <macro name="citation-layout">
+    <choose>
+      <if position="ibid-with-locator">
+        <text term="ibid"/>
+        <text macro="locator" prefix="，"/>
+      </if>
+      <else-if position="ibid">
+        <text term="ibid"/>
+      </else-if>
+      <else-if position="subsequent">
+        <choose>
+          <if disambiguate="true">
+            <group delimiter="，">
+              <text variable="first-reference-note-number" prefix="同前注〔" suffix="〕"/>
+              <group delimiter="：">
+                <text macro="author-short"/>
+                <text variable="title" form="short" prefix="《" suffix="》"/>
+              </group>
+              <text macro="locator"/>
+            </group>
+          </if>
+          <else>
+            <group delimiter="，">
+              <text term="op-cit"/>
+              <text macro="author-short" suffix="书"/>
+              <text macro="locator"/>
+            </group>
+          </else>
+        </choose>
+      </else-if>
+      <else>
+        <text macro="source"/>
+      </else>
+    </choose>
+  </macro>
+  <macro name="citation-layout-en">
```

Comment:
this macro is not used anymore and is giving you the validation error. Delete if indeed is not needed anymore.